NDTensors/ext/NDTensorsCUDAExt/imports.jl

```diff
@@ -1,4 +1,5 @@
 import NDTensors: cu, similartype
 import NDTensors:
   ContractionProperties, _contract!, GemmBackend, auto_select_backend, _gemm!, iscu
-import NDTensors.SetParameters: nparameters, get_parameter, set_parameter, default_parameter
+import NDTensors.TypeParameterAccessors:
+  nparameters, get_parameter, set_parameter, default_parameter
```

Comment:
```suggestion
  nparameters, parameter, set_parameter, default_parameter
```

Reply:
These aren't even actually  used right now because `set_types.jl` isn't necessary so I removed them. There is definitely a lot of cleaning that is needed so I will look through all of the directories. Thanks!